src/commands/build-information.yml

```diff
@@ -107,7 +112,9 @@ steps:
           BuildNumber: "$CIRCLE_BUILD_NUM",
           BuildUrl: "$CIRCLE_BUILD_URL",
           Branch: "$CIRCLE_BRANCH",
-          VcsType: "GitHub"
+          VcsType: "<< parameters.vcs_type >>",
```

Comment:
You can probably make this dynamic, too, by using `<< pipeline.project.type >>`. See https://circleci.com/docs/2.0/variables#pipeline-values.